aioshelly/const.py

```diff
@@ -46,7 +47,10 @@
     # Gen2 RPC based models
     "SNSW-001P16EU": "Shelly Plus 1PM",
     "SNSW-001X16EU": "Shelly Plus 1",
+    "SNSW-0024X": "Shelly Plus i4",
```

Comment:
We officially call it I4, with a capital `I`, not that it matters much.

Reply:
@kzyapkov In this case, the name in the Shelly Shop is wrong.

![Screenshot_20220228-204535](https://user-images.githubusercontent.com/478555/156048823-592300b4-9573-471a-8255-c2417e3a347b.jpg)